projects/1.16/assets/fantasys-furniture/fantasyfurniture/lang/zh_cn.json

```
@@ -0,0 +1,171 @@
+  "block.fantasyfurniture.decorations.nordic.soul_gems_light": "灵魂宝石",
+  "block.fantasyfurniture.decorations.nordic.sweetrolls": "甜卷",
+  "block.fantasyfurniture.decorations.nordic.sweetrolls.stacked": "堆积的甜卷",
+  "block.fantasyfurniture.decorations.paper_stack": "纸叠",
+  "block.fantasyfurniture.decorations.tankards_empty": "酒杯",
+  "block.fantasyfurniture.decorations.tankards_empty.stacked": "堆积的大啤酒",
+  "block.fantasyfurniture.decorations.tankards_honeymead": "蜂蜜酒",
+  "block.fantasyfurniture.decorations.tankards_honeymead.stacked": "堆积的蜂蜜酒",
+  "block.fantasyfurniture.decorations.tankards_milk": "牛奶啤酒",
+  "block.fantasyfurniture.decorations.tankards_milk.stacked": "堆积的牛奶啤酒",
+  "block.fantasyfurniture.decorations.tankards_sweetberry": "甜莓酒",
+  "block.fantasyfurniture.decorations.tankards_sweetberry.stacked": "堆积的甜莓酒",
+  "block.fantasyfurniture.decorations.venthyr.chalices": "温西尔圣杯",
+  "block.fantasyfurniture.decorations.venthyr.chalices.stacked": "堆积的温西尔圣杯",
+  "block.fantasyfurniture.decorations.venthyr.food_0": "温西尔食品 1",
+  "block.fantasyfurniture.decorations.venthyr.food_1": "温西尔食品 2",
+  "block.fantasyfurniture.decorations.venthyr.platter": "温西尔拼盘",
+  "block.fantasyfurniture.decorations.venthyr.platter.stacked": "堆积的温西尔拼盘",
+  "block.fantasyfurniture.decorations.venthyr.tea_cups": "温西尔茶杯",
+  "block.fantasyfurniture.decorations.venthyr.tea_cups.stacked": "堆积的温西尔茶杯",
+  "block.fantasyfurniture.decorations.venthyr.tea_set": "温西尔茶具",
+  "block.fantasyfurniture.decorations.venthyr.tomes": "温西尔书籍",
+  "block.fantasyfurniture.decorations.venthyr.tomes.stacked": "堆积的温西尔书籍",
+  "block.fantasyfurniture.decorations.venthyr.widow_bloom": "寡妇花瓶",
+  "block.fantasyfurniture.dunmer.bed_double": "邓默双人床",
+  "block.fantasyfurniture.dunmer.bed_double.occupied": "这张床已被占用",
+  "block.fantasyfurniture.dunmer.bed_single": "邓默单人床",
+  "block.fantasyfurniture.dunmer.bed_single.occupied": "这张床已被占用",
+  "block.fantasyfurniture.dunmer.bench": "邓默长凳",
+  "block.fantasyfurniture.dunmer.bench.occupied": "这个座位已被占用",
+  "block.fantasyfurniture.dunmer.bookshelf": "邓默书架",
+  "block.fantasyfurniture.dunmer.carpet": "邓默地毯",
+  "block.fantasyfurniture.dunmer.chair": "邓默椅",
+  "block.fantasyfurniture.dunmer.chair.occupied": "这个座位已被占用",
+  "block.fantasyfurniture.dunmer.chandelier": "邓默枝形吊灯",
+  "block.fantasyfurniture.dunmer.chest": "邓默宝箱",
+  "block.fantasyfurniture.dunmer.cushion": "邓默软垫",
+  "block.fantasyfurniture.dunmer.cushion.occupied": "这个座位已被占用",
+  "block.fantasyfurniture.dunmer.desk_left": "邓默办公桌左",
+  "block.fantasyfurniture.dunmer.desk_right": "邓默办公桌右",
+  "block.fantasyfurniture.dunmer.door_double": "邓默门 2",
+  "block.fantasyfurniture.dunmer.door_single": "邓默门 1",
+  "block.fantasyfurniture.dunmer.drawer": "邓默抽屉",
+  "block.fantasyfurniture.dunmer.dresser": "邓默德莱瑟",
+  "block.fantasyfurniture.dunmer.floor_light": "邓默落地灯",
+  "block.fantasyfurniture.dunmer.painting_small": "邓默绘画小",
+  "block.fantasyfurniture.dunmer.painting_wide": "邓默宽幅绘画",
```

```suggestion
  "block.fantasyfurniture.dunmer.painting_wide": "宽幅丹默绘画",
```
应将邓莫改为丹莫（前文提过原因），同时，将绘画改为小幅xx绘画和宽幅xx绘画会在句式同一性以及语序上更恰当